### PR TITLE
WT-6670 Fix uninitialized buffer

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: test/csuite/wt6616_checkpoint_oldest_ts/main.c [hunk 1]
```diff
@@ -139,6 +139,7 @@ thread_run(void *arg)
     char kname[64], tscfg[64];
```
Comment (raviprakashgiri29) [inline test/csuite/wt6616_checkpoint_oldest_ts/main.c:139]: Can you memset `tscfg` buffer this as well?

Comment (quchenhao) [reply test/csuite/wt6616_checkpoint_oldest_ts/main.c:139]: `tscfg` is fine. We only read the part which is written as it is a string.

Comment (quchenhao) [reply test/csuite/wt6616_checkpoint_oldest_ts/main.c:139]: kanme is different as we are treating it as binary data.

**Review by raviprakashgiri29 — APPROVED** [review]
LGTM.

File: test/csuite/wt6616_checkpoint_oldest_ts/main.c (hunk 1 above)
Comment (raviprakashgiri29) [reply test/csuite/wt6616_checkpoint_oldest_ts/main.c:139]: Yeah, that makes sense. Thank you.

**Pull request merged**